var/spack/repos/builtin/packages/nek5000/package.py

```diff
@@ -93,101 +57,22 @@ def install(self, spec, prefix):
 
         fflags = spec.compiler_flags['fflags']
         cflags = spec.compiler_flags['cflags']
-        if ('+prenek' in spec) or ('+postnek' in spec):
-            libx11_h = find_headers('Xlib', spec['libx11'].prefix.include,
-                                    recursive=True)
-            if not libx11_h:
-                raise RuntimeError('Xlib.h not found in %s' %
-                                   spec['libx11'].prefix.include)
-            cflags += ['-I%s' % os.path.dirname(libx11_h.directories[0])]
-
-            xproto_h = find_headers('X', spec['xproto'].prefix.include,
-                                    recursive=True)
-            if not xproto_h:
-                raise RuntimeError('X.h not found in %s' %
-                                   spec['xproto'].prefix.include)
-            cflags += ['-I%s' % os.path.dirname(xproto_h.directories[0])]
-
-            libxt_h = find_headers('Intrinsic', spec['libxt'].prefix.include,
-                                   recursive=True)
-            if not libxt_h:
-                raise RuntimeError('X11/Intrinsic.h not found in %s' %
-                                   spec['libxt'].prefix.include)
-            cflags += ['-I%s' % os.path.dirname(libxt_h.directories[0])]
+
         if self.compiler.name in ['xl', 'xl_r']:
             # Use '-qextname' to add underscores.
             # Use '-WF,-qnotrigraph' to fix an error about a string: '... ??'
             fflags += ['-qextname', '-WF,-qnotrigraph']
-        fflags = ' '.join(fflags)
-        cflags = ' '.join(cflags)
 
-        # Build the tools, maketools copy them to Nek5000/bin by default.
-        # We will then install Nek5000/bin under prefix after that.
-        with working_dir(tools_dir):
-            # Update the maketools script to use correct compilers
-            filter_file(r'^#FC\s*=.*', 'FC="{0}"'.format(fc), 'maketools')
-            filter_file(r'^#CC\s*=.*', 'CC="{0}"'.format(cc), 'maketools')
-            if fflags:
-                filter_file(r'^#FFLAGS=.*', 'FFLAGS="{0}"'.format(fflags),
-                            'maketools')
-            if cflags:
-                filter_file(r'^#CFLAGS=.*', 'CFLAGS="{0}"'.format(cflags),
-                            'maketools')
+        error = Executable(fc)('this-is-so-dumb.f', output=str, error=str,
```

Comment:
Let's use a more professional name here, if there isn't a nicer way to test for this (perhaps by gfortran version)?

Reply:
@thilinarmtb, can you address this please?